adding Arduino UNO Wifi rev02

andrewjohnsen31 opened the pull request:
added defined " AVR_ATmega4809"

tested and verified

andrewjohnsen31 commented:
Sorry, I was unable to perform clang-format.
Spent 2 days, If you could help it will be great.
For now I have to give it a break

Andrew

> On Apr 23, 2020, at 11:40 AM, ladyada <notifications@github.com> wrote:
> 
> 
> @ladyada requested changes on this pull request.
> 
> https://github.com/adafruit/Adafruit_TouchScreen/pull/29/checks?check_run_id=612387705 <https://github.com/adafruit/Adafruit_TouchScreen/pull/29/checks?check_run_id=612387705>
> please run clang-format or perform the fixes in the CI error report
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/adafruit/Adafruit_TouchScreen/pull/29#pullrequestreview-399228277>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AOE5NZSKCLRXMX3A7HYLQRTROBOPZANCNFSM4MPELC5A>.
> 

Andrew Johnsen
andrewjohnsen31w@gmail.com
(603) 718-0745

hathach commented:
It is relatively easy if you have clang-format installed. Just run

```
clang-format -i *.cpp
```

or just `*` for all files. clang will perform in-place replace to all the files and you are good to commit.

andrewjohnsen31 commented:
Thanks,

The only file I changed was: touchscreen.h
The Error report also showed the my changes as additions and subtractions.

What I did
In terminal:
Changed directory to where I put touchscreen.h no other files.
Clang-format -i touchscreen.h

Clang-format did not report and error and the file was not updated.
Is there a report out file.

I really appreciate the help.

I don’t understand how Clang-format finds the file.

At home retired, I have no excuse for not spending more time. :)

Andrew Johnsen
‭+1 (603) 718-0745‬

> On Apr 25, 2020, at 9:54 AM, Ha Thach <notifications@github.com> wrote:
> 
> ﻿
> It is relatively easy if you have clang-format installed. Just run
> 
> clang-format -i *.cpp
> or just * for all files. clang will perform in-place replace to all the files and you are good to commit.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.

hathach commented:
> Thanks, The only file I changed was: touchscreen.h The Error report also showed the my changes as additions and subtractions. What I did In terminal: Changed directory to where I put touchscreen.h no other files. Clang-format -i touchscreen.h Clang-format did not report and error and the file was not updated. Is there a report out file. I really appreciate the help. I don’t understand how Clang-format finds the file. At home retired, I have no excuse for not spending more time. :) Andrew Johnsen ‭+1 (603) 718-0745‬
> […](#)
> On Apr 25, 2020, at 9:54 AM, Ha Thach ***@***.***> wrote: ﻿ It is relatively easy if you have clang-format installed. Just run clang-format -i *.cpp or just * for all files. clang will perform in-place replace to all the files and you are good to commit. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

If you run it, then it already re-format that file, you should see the different in git status. Try to commit and push it up, it should pass the CI test now.

andrewjohnsen31 commented:
Some success, CI : Passed, Clang-Format : Passed
Now I am stuck on Pushing. 
Do I have to do it in terminal window ?
1 open Item: Ladyada was requested for review. (My mistake in requesting)
